docs/integrations/sources/metabase.md

```diff
@@ -12,6 +12,9 @@ This Source is capable of syncing the following Streams:
 * [Card](https://www.metabase.com/docs/latest/api/card.html#get-apicard)
 * [Collections](https://www.metabase.com/docs/latest/api/collection.html#get-apicollection)
 * [Dashboard](https://www.metabase.com/docs/latest/api/dashboard.html#get-apidashboard)
+* [Database](https://www.metabase.com/docs/latest/api/database.html#get-apidatabase)
+* Dataset Query Views: This requires an admin account to a metabase account with at least a PRO plan to retrieve data access audit logs.
```

Comment:
Do we have a retry strategy/raise exception warnings for the users that are not on the PRO plan, to skip such streams?

Reply:
No but i'm not aware on how to implement that...

Reply:
we can override default CDK HTTP `should_retry` method and check the response object for certain error codes there, like here: https://github.com/airbytehq/airbyte/commit/50014742f38a60dbce86898e270f109b0974c7fb#diff-72e6ac82d1e9829b12bc0de4721cf47955aa9d72a843b2efb0526f898a3be9b9R69